src/base/bittorrent/session.cpp

```diff
@@ -786,11 +787,14 @@ bool Session::removeCategory(const QString &name)
     if (isSubcategoriesEnabled()) {
         // remove subcategories
         QString test = name + "/";
-        foreach (const QString &category, m_categories.keys()) {
+        for (auto i = m_categories.begin(); i != m_categories.end(); ) {
+            const QString category = i.key();
             if (category.startsWith(test)) {
-                m_categories.remove(category);
+                i = m_categories.erase(i);
                 result = true;
                 emit categoryRemoved(category);
+            } else {
```

Comment:
newline before `else`

Reply:
@Chocobo1 Can't understand this. What new line ?

Reply:
Add newline before `else`